tests/unit/header/Header.spec.js

```diff
@@ -0,0 +1,9 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
Please put the components tests under a components folder
`tests/unit/components/header/Header.spec.js`